plugin/http/httpstats/transport.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httpstats
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/tag"
+)
+
+// Transport is an http.RoundTripper that collects stats for the outgoing requests.
+type Transport struct {
+	// Base represents the underlying roundtripper that does the actual requests.
+	// If none is given, http.DefaultTransport is used.
+	//
+	// If base HTTP roundtripper implements CancelRequest,
+	// the returned round tripper will be cancelable.
+	Base http.RoundTripper
+}
+
+// RoundTrip implements http.RoundTripper, delegating to Base and recording stats for the request.
+func (t Transport) RoundTrip(req *http.Request) (*http.Response, error) {
+	ctx, _ := tag.New(req.Context(),
```

Comment:
Let's not ignore this error.

Reply:
There's not really anything that I can do at this point. Ignoring it seemed like the best option. What would you suggest?

Reply:
So the thing is that if say the Path is greater than 255
https://github.com/census-instrumentation/opencensus-go/blob/c43bafa4c06b74700a7268922c207bb3f5c55d33/tag/validate.go#L20:2
 or so we'll have a problem and it won't get inserted.
But for sure, nothing more that we can do :)